src/main/groovy/life/qbic/WriterHelper.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic
+
+import java.nio.file.Path
+
+/**
+ * <add class description here>
```

Comment:
> `<add class description here>`

Reply:
Why do you need this class? `writeToFileSystem` sounds like it should be in the `FileSystemWriter`